### PR TITLE
Timestamps

### PR DESCRIPTION
**Pull request by burde041**

### CODE REVIEW

File: client/src/app/owner/owner.component.scss [hunk 1]
```diff
@@ -12,6 +12,12 @@
   flex-grow: 1;
 }
 
+#date{
```
Comment (audreylemeur) [inline client/src/app/owner/owner.component.scss:15]: Since this is on both owner and viewer pages, we could add this to note.scss instead

**Comment by audreylemeur** [comment]
I pulled this branch down to refactor the css and I can't get any notes to appear due to server error.

File: client/src/app/owner/owner.component.scss (hunk 1 above)
Comment (burde041) [reply client/src/app/owner/owner.component.scss]: Just did this, good catch.

**Comment by ErikRauer** [comment]
> I pulled this branch down to refactor the css and I can't get any notes to appear due to server error.

Without knowing any more details my best guess is that you need to clear the already existing notes from your database. The fact that they don't have any dates on them probably causes the sorting to throw an error.

**Comment by ErikRauer** [comment]
Could you fix the lgtm alerts? I think just changing the ids that it mentions into classes should do the trick.

**Pull request merged**